tox.ini

```diff
@@ -1,9 +1,8 @@
 [tox]
-envlist = py38-django{32,42}-celery{44,50}-drf{latest}
+envlist = py{38,312}-django{42}-celery{50}-drf{latest}
```

Comment:
Need to add `py311` to the env list here so we test Python 3.11 properly.

Reply:
All changes fixed.